hpke_test.go

```diff
@@ -12,25 +14,23 @@ var (
 	original = []byte("Beauty is truth, truth beauty")
 	aad      = []byte("that is all // Ye know on earth, and all ye need to know")
 	info     = []byte("Ode on a Grecian Urn")
-	rtts     = 10
+	rtts     = 1
 )
 
 func roundTrip(t *testing.T, kemID KEMID, kdfID KDFID, aeadID AEADID, enc *EncryptContext, dec *DecryptContext) {
-	for range make([]struct{}, rtts) {
-		encrypted := enc.Seal(aad, original)
-		decrypted, err := dec.Open(aad, encrypted)
-		if err != nil {
-			t.Fatalf("[%x, %x, %x] Error in Open: %s", kemID, kdfID, aeadID, err)
-		}
+	encrypted := enc.Seal(aad, original)
```

Comment:
Why did you remove the multi-RTT version?

Reply:
That's a leftover change I forgot to revert. I'll bring it back. (Previously when the vector generation was woven through the tests this multi-RT variant complicated things, so I dropped it.)